.gitignore

```diff
@@ -190,3 +190,15 @@ Session.vim
 
 # VS debug support files
 launchSettings.json
+
+# Swift output files
+*.swiftinterface*
+*.swiftdoc*
+*.swiftmodule*
+*.abi*
+*.swiftsourceinfo*
+*.dylib
+
+# Testing artifacts
+testing/
+src/samples/**/*Bindings.cs
```

Comment:
These should all go under `artifacts/`. In the current experiment form, this seems okay, but prior to merging into dotnet/runtime, all outputs should be under `artifacts/`.

Reply:
Oops. Sorry, I thought we were working in a dotnet/runtime fork. Never mind.